champions/src/main/java/me/mykindos/betterpvp/champions/champions/skills/skills/brute/data/BlockTossObject.java

```diff
@@ -101,14 +106,21 @@ public void spawn(double size) {
 
             // Get a random translation so the boulder looks more natural
             // Center the block display by subtracting 0.5, acts as our reference point for offsets
-            float xTranslation = (float) ((Math.random()) - 0.5f - 0.5f);
-            float yTranslation = (float) ((Math.random()) - 0.5f) - 0.5f;
-            float zTranslation = (float) ((Math.random()) - 0.5f - 0.5f);
+            float xTranslation = (float) ((Math.random()) - 0.5f) * displacement;
+            float yTranslation = (float) ((Math.random()) - 0.5f) * displacement;
+            float zTranslation = (float) ((Math.random()) - 0.5f) * displacement;
```

Comment:
This actually isn't a displacement, this makes sure that the displacement is absolute and not relative. Display blocks' locations are shifted by 0.5 in each direction, so your effective displacement here is `displacement + 0.5` instead of just `displacement`.

Reply:
this is OK, that was my intention and it works as I desired